### PR TITLE
make-dist: generate ceph.spec

### PR DESCRIPTION
**Pull request by liewegas**
Signed-off-by: Sage Weil <sage@redhat.com>

### CODE REVIEW

File: make-dist [hunk 1]
```diff
@@ -31,10 +31,18 @@ bin/git-archive-all.sh --prefix ceph-$version/ \
...
 		       $outfile.tar
 
-echo "including src/.git_version and src/ceph_ver.h..."
+# populate files with version strings
+echo "including src/.git_version, src/ceph_ver.h, ceph.spec"
 src/make_version -g src/.git_version -c src/ceph_ver.h
+
+rpm_version=`echo $version | cut -d - -f 1-1`
+rpm_release=`echo $version | cut -d - -f 2- | sed 's/-/./'`
+cat ceph.spec.in | \
```
Comment (tchaikov) [inline make-dist]: ahh, i have a PR for this also #9978

**Comment by tchaikov** [comment]
could have ```Fixes: http://tracker.ceph.com/issues/16501``` in the commit message.

**Comment by dmick** [comment]
Test failure is irrelevant, and happened on trusty, so, ignoring.

**Comment by dmick** [comment]
Should resolve http://tracker.ceph.com/issues/16501

**Pull request merged**